src/cli.rs

```diff
@@ -243,5 +256,16 @@ pub fn extract_args(matches: &ArgMatches) -> Config {
     args.txs_batch_size = matches
         .value_of("batch-size")
         .map(|batch_size_str| batch_size_str.parse().expect("can't parse batch-size"));
+
+    let (_, kp_auth_path) = ConfigInput::compute_keypair_path_setting(
+        matches.value_of("keeper-authority").unwrap_or(""),
+        &config.keypair_path,
+    );
+
+    if let Ok(kpa) = read_keypair_file(kp_auth_path) {
```

Comment:
in case if you want to use identity of an existing validator for your client which is on the machine with another IP address, I guess you will need bind-address. Otherwise probably not